package.json

```diff
@@ -10,7 +10,9 @@
     "typescript"
   ],
   "license": "MIT",
-  "name": "ts-workflow",
+  "name": "@fewlines/ts-workflow",
+  "main": "dist/index.js",
+  "types": "index.d.ts",
```

Comment:
Shouldn't we commit this file? Moreover the `declarationDir` is set to `./dist` so I don't understand why it's not in `./dist/index.d.ts` ?

Reply:
> Shouldn't we commit this file? 
nop this is published but not commit (if you want I can link guides I found that specified to do it this way) 

> Moreover the declarationDir is set to ./dist so I don't understand why it's not in ./dist/index.d.ts ?
Nice catch the path is wrong. 👍